module-services/service-cellular/ServiceCellular.cpp

```diff
@@ -2194,9 +2196,9 @@ auto ServiceCellular::handleCellularRingNotification(sys::Message *msg) -> std::
         return std::make_shared<CellularResponseMessage>(hangUpCall());
     }
 
+    /// NOTE: the code below should be investigated as there is something weird in this flow
```

Comment:
for some weird reason, we are treating RING as CLIP in here. Though currently without this messaging we have automatic call denial.